synapse/handlers/room_member.py

```diff
@@ -1138,6 +1138,11 @@ async def _make_and_store_3pid_invite(
         if room_name_event:
             room_name = room_name_event.content.get("name", "")
 
+        room_type = ""
+        room_create_event = room_state.get((EventTypes.Create, ""))
+        if room_create_event:
+            room_type = room_create_event.content.get("type", "")
```

Comment:
I kept using the wrong field here so I ended up making a constant...

```suggestion
            room_type = room_create_event.content.get(EventContentFields.ROOM_TYPE, "")
```

Reply:
Great, thanks!